### PR TITLE
blockdev_full_backup_invalid_sync_mode:update error msg

### PR DESCRIPTION
**Pull request by aliang123**
Update error msg since qemu6.2

Signed-off-by: Aihua Liang <aliang@redhat.com>
id:2034080

### CODE REVIEW

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup_invalid_sync_mode.q35: PASS (106.54 s)

**Comment by aliang123** [comment]
qemu6.0.0:
   (1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup_invalid_sync_mode.q35: PASS (44.34 s)

**Comment by aliang123** [comment]
@zhencliu @ngu-niny, please help to reveiw, thanks.

File: qemu/tests/cfg/blockdev_full_backup_invalid_sync_mode.cfg [hunk 1]
```diff
@@ -21,3 +21,5 @@
     rebase_mode = unsafe
     qmp_error_msg = "Invalid parameter '%s'"
     full_backup_options = '{"sync": "aaa"}'
+    required_qemu_version = [6.2.0, )
+    qmp_error_msg_new = "Parameter 'sync' does not accept value '%s'"
```
Comment (zhencliu) [inline qemu/tests/cfg/blockdev_full_backup_invalid_sync_mode.cfg:25]: We can add the qemu version as part of the param name, just as what you did this before(e.g. blockdev_stream_to_invalid_node.cfg):
qmp_error_before_6_2
qmp_error_since_6_2

**Comment by zhencliu** [comment]
LGTM

**Comment by ngu-niny** [comment]
Test PASS on both old and new qemu versions:
 
(1/1) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_full_backup_invalid_sync_mode: PASS (104.75 s)
(1/1) Host_RHEL.m9.u0.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.blockdev_full_backup_invalid_sync_mode.s390-virtio: PASS (26.05 s)

Acked-by: Nini Gu <ngu@redhat.com>

**Pull request merged**